Resources/Private/Fusion/Shame.fusion

```diff
@@ -0,0 +1,3 @@
+##
+# Quick & Dirty fixes, that can be improved later on
+#
```

Comment:
Check https://www.youtube.com/watch?v=8bZikINA2UA&feature=youtu.be most of the best practice come from this talk